src/raftstore/errors.rs

```diff
@@ -166,7 +166,14 @@ impl Into<errorpb::Error> for Error {
                     .mut_raft_entry_too_large()
                     .set_entry_size(entry_size);
             }
-            Error::StoreNotMatch(..) => errorpb.set_store_not_match(errorpb::StoreNotMatch::new()),
+            Error::StoreNotMatch(to_store_id, my_store_id) => {
+                errorpb
+                    .mut_store_not_match()
+                    .set_expect_store_id(to_store_id);
```

Comment:
Should `expect_store_id` be `my_store_id`?

Reply:
How about `requested_store_id` and `actual_store_id`?

Reply:
Or `expect_store_id` and `actual_store_id`?

Reply:
expect_store_id and actual_store_id +1

Reply:
I think `request_store_id` is better. By `expect`, I connot know it is the client or the server that is expecting.